### ISSUE TITLE
Are Websockets working

### ISSUE THREAD

**Issue by StyleASD** [cobinhood/api-public#9]
Hi,

I am trying to have a play with the API and am unable to connect to the websocket address provided. The error message I am currently getting is error: Error: unexpected server response (501).

I have confirmed this from two different sources. One using the gorilla/websocket in go and the other using an npm lib wscat  'wscat -c wss://feed.cobinhood.com'

Kr

Aled

**Comment by DSoane** [comment]
+1

I am having the exact same issue, Aled. And I'm using C++ to write raw.

Dave.

**Comment by MHamburg** [comment]
Does connecting to wss://feed.cobinhood.com/ws work for you?

**Comment by DSoane** [comment]
Yes! Thank you!

It now works (well, it upgrades to websocket, I haven't tested further), on the live site. It still gives error 501 on the sandbox site.

I had actually tried this before, but I am using OpenSSL for the comms, and I added the /ws in the host I passed to BIO_do_connect(), not realising that that must ONLY be the host, and you must add the /ws after the GET when you request the upgrade to websockets.

I'm learning every day!

Dave.
